src/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Apr 26 12:53:22 EDT 2017
+4=\#
+3=org.springframework
+2=
+1=javax
+0=java
```

Comment:
I can't understand the content of this class, but somehow it doesn't work for me:

![image](https://user-images.githubusercontent.com/1071590/26951673-090f9322-4c70-11e7-9b9b-27965821b1c9.png)



When I perform reformat in on of the classes the `org.springframework` imports go exactly after all other `org` like in Spring Integration style. For example:

![image](https://user-images.githubusercontent.com/1071590/26951755-5bdb6acc-4c70-11e7-9217-3ad36f555870.png)


instead of proposed by you here in PR:


![image](https://user-images.githubusercontent.com/1071590/26951775-70641e6c-4c70-11e7-8949-f58d99d5ebe2.png)

Thanks